DQMOffline/L1Trigger/src/L1TEGammaOffline.cc

```diff
@@ -759,5 +807,30 @@ void L1TEGammaOffline::bookPhotonHistos(DQMStore::IBooker & ibooker)
   ibooker.cd();
 }
 
+void L1TEGammaOffline::endJob(){
+  normalise2DHistogramsToBinArea();
+}
+
+void L1TEGammaOffline::normalise2DHistogramsToBinArea() {
+  std::vector<MonitorElement *> monElementstoNormalize = {
+      h_L1EGammaETvsElectronET_EB_,    h_L1EGammaETvsElectronET_EE_,
+      h_L1EGammaETvsElectronET_EB_EE_, h_L1EGammaPhivsElectronPhi_EB_,
+      h_L1EGammaPhivsElectronPhi_EE_,  h_L1EGammaPhivsElectronPhi_EB_EE_,
+      h_L1EGammaEtavsElectronEta_,     h_L1EGammaETvsPhotonET_EB_,
+      h_L1EGammaETvsPhotonET_EE_,      h_L1EGammaETvsPhotonET_EB_EE_,
+      h_L1EGammaPhivsPhotonPhi_EB_,    h_L1EGammaPhivsPhotonPhi_EE_,
+      h_L1EGammaPhivsPhotonPhi_EB_EE_, h_L1EGammaEtavsPhotonEta_
+  };
+
+  for (auto mon : monElementstoNormalize) {
+    if (mon != nullptr) {
+      auto h = mon->getTH2F();
+      if (h != nullptr) {
+        h->Scale(1, "width");
```

Comment:
So the "width" option also works properly for 2d histograms in the end?

Reply:
Yes, it calls a special case of `Add`: 
 - https://root.cern.ch/doc/master/classTH1.html#a1913bea50293a57b9ee65bf7f65a26e1
   - ANOTHER SPECIAL CASE : h1 = h2 and c2 < 0 do a scaling this = c1 * h1 / (bin Volume)